_includes/pbc/all/install-features/202304.0/install-warehouse-user-management-feature.md

```diff
@@ -0,0 +1,319 @@
+
+
+
+This document describes how to integrate the Warehouse User Management feature into a Spryker project.
```

Comment:
Why is this feature absent in the Release Groups app?

Reply:
It is a new feature, it will appear there after the release.